asteroid/masknn/blocks.py

```diff
@@ -446,9 +446,17 @@ def __init__(self, in_chan, n_src, rnn_type = 'lstm',
                 in_chan * embedding_dim)
         self.mask_layer = nn.Linear(rnn_out_dim, in_chan * n_src)
         self.non_linearity = nn.Sigmoid()
+        self.log = log
+        self.EPS = torch.finfo(torch.float32).eps
+        if log:
+            #TODO: Use pytorch lightning logger here
+            print('Using log spectrum as input')
```

Comment:
It's fine not to print anything actually, it will be be printed in the conf dictionary. 
I would call it differently though, maybe `take_log` or `log_spec`?